gp-includes/things/translation-set.php

```diff
@@ -314,13 +316,13 @@ public function import( $translations, $desired_status = 'current' ) {
 				/**
 				 * Filter the the status of imported translations of a translation set.
 				 *
-				 * @since 1.0.0
+				 * @since 2.3.0
```

Comment:
The filter has been in place since 1.0, this shouldn't be changed?  Maybe add a comment about the additional parameter in the comments above?